### PR TITLE
[Wrapper]: RecordEpisodeStatistics

### PR DESCRIPTION
**Pull request by zuoxingdong**

### CODE REVIEW

File: gym/wrappers/record_episode_statistics.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+import gym
+
+
+class RecordEpisodeStatistics(gym.Wrapper):
+    def __init__(self, env, deque_size=100):
+        super(RecordEpisodeStatistics, self).__init__(env)
+        self.t0 = time.time()  # TODO: use perf_counter when gym removes Python 2 support
+        self.episode_return = 0.0
+        self.episode_horizon = 0
```
Comment (pzhokhov) [inline gym/wrappers/record_episode_statistics.py]: would episode_length not be more descriptive?

**Comment by pzhokhov** [comment]
this seems very similar to baselines Monitor class (except for saving data to a file); however, uses completely different names and so is incompatible. @zuoxingdong do you think it makes sense to change the info dict keys for the sake of compatibility?

File: gym/wrappers/record_episode_statistics.py (hunk 1 above)
Comment (zuoxingdong) [reply gym/wrappers/record_episode_statistics.py]: @pzhokhov Indeed, it is more descriptive. Thanks!

**Comment by zuoxingdong** [comment]
@pzhokhov Thanks a lot for your comments. I've made the change accordingly.

**Pull request merged**